layouts/partials/head.html

```diff
@@ -6,5 +6,14 @@
 		{{ range .Site.Menus.main }}
 		<a href="{{ .URL }}">{{ .Name }}</a>
 		{{ end }}
+
+		<div class="toggle-switch">
+			<input type="checkbox" id="chkTest" name="chkTest" onchange="toggleDarkMain()">
```

Comment:
Better to add the function as a listener instead of directly passing it to onchange

Reply:
Done 💯